stripper/ze_last_man_standing_p3.cfg

```diff
@@ -218,34 +646,8 @@ modify:
 	}
 }
 
-// Optional: Prevent the "Portal Gun" skip.
-//modify:
-//{
-//	match:
-//	{
-//		"classname" "func_button"
-//		"targetname" "Weapon_PortalGun_UI"
-//	}
-//	replace:
-//	{
-//		"origin" "-1004 13248 -8512"
-//	}
-//}
-
-//modify:
-//{
-//	match:
-//	{
-//		"classname" "env_entity_maker"
-// 		"targetname" "Weapon_PortalGun_Projectile_Spawner"
-//	}
-//	replace:
-//	{
-//		"origin" "-1088 13248 -8520"
-//	}
-//}
```

Comment:
I don't see the reason for removing this. We've currently chosen to disable it (comment it out), however that doesn't mean we may not choose to use it in the future.

Reply:
Ok.